Automattic/jetpack
# Add persistent "link to wpcom" notice when user not linked (and other notice improvements)

This adds a persistent notice that will be shown to unlinked **users**, similar to the button that is shown in current Jetpack at the top of the page. 

It looks like this: 
![unlinked](https://cloud.githubusercontent.com/assets/7129409/17627854/bf1c6622-6080-11e6-92f2-f5447802d279.png)

It's not dismissible. 

**To Test:** 
- Log into a connected Jetpack site as any unlinked user.  
- You should be able to link to WordPress.com
- You should not see the notice when you are linked
- You should not see the notice when the site is not connected

**Other Improvements to notices**
- Added README about different types of notices we're using.
- Now using proper `NoticeAction` for notice CTA's, rather than inline links.  
- Looks much better: 

Before:
![before-1](https://cloud.githubusercontent.com/assets/7129409/17627896/e5f2caac-6080-11e6-89b3-47eec5386126.png)
![before-2](https://cloud.githubusercontent.com/assets/7129409/17627899/e7e5e420-6080-11e6-995d-9a89f94df1ab.png)

After: 
![after-1](https://cloud.githubusercontent.com/assets/7129409/17627903/eb743a88-6080-11e6-9fa4-401029ecbcb0.png)
![after-2](https://cloud.githubusercontent.com/assets/7129409/17627904/ee41c24e-6080-11e6-8e78-a0782324a9d0.png)
